modules/learn/pages/security/on-the-wire-security.adoc

```diff
@@ -37,3 +38,85 @@ Once the selection has been made by Couchbase Server, Couchbase Server notifies
+
+* If a custom list of cipher-suites is established for a given service, the service uses that list.
+
+* If no custom list of cipher-suites is established for a given service, but a global list is established, the service uses the global list.
+
+* If no custom list of cipher-suites is established for a given service, and no global list is established, the service uses its pre-configured list of supported cipher-suites.
+
+Any cipher-suites included in a per-service or global list that do not appear on the pre-configured list of supported cipher-suites for that service are ignored by the service.
+
+There is no _global_, pre-configured list of supported cipher-suites.
+
+Cipher-suites are attemptedly used by a service in the order in which the cipher-suites appear in the list established for the service.
+The order of cipher-suites in a custom list may be varied from that of the pre-configured list of supported cipher-suites for the service, except in the case of 1.3 cipher-suites when used by the _Index_, _Search_, _Query_ and _Backup_ services: see xref:learn:security/on-the-wire-security.adoc#cipher-suite-configuration-limitations[Cipher-Suite Configuration Limitations], below, for information.
+
+[#cipher-suite-configuration-limitations]
+== Cipher-Suite Configuration Limitations
+
+The following limitations apply to cipher-suite configuration.
+
+[#tls-3-cipher-suite-limitations]
+=== TLS 1.3 Cipher-Suite Limitations
+
+As indicated above, only the _Data_, _Search_, _Index_, _Query_, _Eventing_, _Analytics_, and _Backup_ services can use TLS 1.3 cipher-suites.
+TLS 1.3 cipher-suites _cannot_ be used by the Cluster Manager.
+
+Given that the _custom-configuration_ of ciphers means the selection of a subset of available ciphers for a custom cipher-suite list, and the specifying of the selected ciphers in the subset in any order judged appropriate; of the services that _can_ use TLS 1.3 ciphers, only the _Data_ and _Analytics_ services support the custom-configuration of TLS 1.3 ciphers.
+The remaining services that can use TLS 1.3 &#8212; _Search_, _Index_, _Query_, and _Backup_ &#8212; do _not_ support custom-configuration of TLS 1.3 ciphers.
+
+This means that a custom cipher-suite list configured for the _Search_, _Index_, _Query_, or _Backup_ service will always implicitly include _all_ the TLS 1.3 cipher-suites in the list of supported cipher-suites for that service, in the order specified: none of those TLS 1.3 cipher-suites can be omitted, nor can their order be changed with respect to one another.
+
+If the minimum TLS version for one of these services is less than 1.3, and only selected non-1.3 cipher-suites are explicitly included in a custom list configured for the service, all 1.3 cipher-suites supported for the service are implicitly appended to the explicitly stated list, in the appropriate order for 1.3 cipher-suites.
```

Comment:
Please remove text about minimum TLS version, it's just confusing. 
Actually there is no extra information that is conveyed by this sentence, that is not accomplished by the previous paragraph.